app/models/link.rb

```diff
@@ -1,2 +1,5 @@
 class Link < ApplicationRecord
+  def title
+    Rails.configuration.open_graph_lib.new(url).title
```

Comment:
This feels a little tightly coupled to the Rails configuration. At this very least, we could inject the open graph lib into this object rather than referencing all the way back to Rails' config. Even better would be to have a helper or decorator that could perform this function for us :dizzy: